### PR TITLE
Simple Legofy GUI 

### PR DESCRIPTION
**Pull request by CasualBeer**
Next step: Binary executable

### CODE REVIEW

**Comment by JuanPotato** [comment]
But why not a way to call it yet?

**Comment by CasualBeer** [comment]
@JuanPotato 

Just run legofy_gui.py :) 

or:
'''
from legofy.legofy_gui import LegofyGui

mygui = LegofyGui()
mygui.mainloop()
'''

**Comment by JuanPotato** [comment]
I was thinking more along something that anyone can run immediately. I guess we can do that later.

**Comment by CasualBeer** [comment]
@JuanPotato 
Yep. Later

**Comment by CasualBeer** [comment]
@JuanPotato 

Maybe click option ? --gui or something like that ?

**Comment by nabijaczleweli** [comment]
Oh god the lack of XPManifest in the screenshot.
I require mental help now.

**Comment by CasualBeer** [comment]
@nabijaczleweli 

Than go get it :)

In the meantime is there anything you or I can do do fix it ? Is there any way to force tkinter to use native windows theme ?

**Comment by nabijaczleweli** [comment]
The dude in [this](http://stackoverflow.com/questions/19245941/adding-a-scrollbar-to-a-frame-using-tkinter-python) question has screenshots, which show his windows with XPManifest on.

**Comment by nabijaczleweli** [comment]
I'd also recommend the window to be non-resizable, because I was able to do this:
small window

File: legofy/legofy_gui.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+from tkinter import filedialog
+import tkinter.messagebox as tkmsg
+
+LEGO_PALETTE = ('none', 'solid', 'transparent', 'effects', 'mono', 'all', )
+
+class LegofyGui(tk.Tk):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.wm_title("Legofy!")
+        self.iconbitmap('./assets/brick.ico')
```
Comment (nabijaczleweli) [inline legofy/legofy_gui.py]: This doesn't work when invoked from any other directory, than the one containing the script.

File: legofy/legofy_gui.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+
+    def choose_a_file(self):
+
+        options = {}
+        options['defaultextension'] = '.jpg'
+        options['filetypes'] = [('JPEG', '.jpg'),
+                                ('GIF', '.gif'),
+                                ('PNG', '.png'),]
+        options['initialdir'] = 'C:\\'
```
Comment (nabijaczleweli) [inline legofy/legofy_gui.py]: No worky on non-Windooze

Comment (JuanPotato) [inline legofy/legofy_gui.py]: fixed

**Pull request merged**